store/s3storeKMS.go

```diff
@@ -0,0 +1,486 @@
+package store
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/pkg/errors"
+)
+
+// latest is used to keep a single object in s3 with all of the
+// most recent values for the given service's secrets.  Keeping this
+// in a single s3 object allows us to use a single s3 GetObject
+// for ListRaw (and thus chamber exec).
+type LatestIndexFile struct {
+	Latest map[string]LatestValue `json:"latest"`
+}
+
+type LatestValue struct {
+	Version  int    `json:"version"`
+	Value    string `json:"value"`
+	KMSAlias string `json:"KMSAlias"`
+}
+
+var _ Store = &S3KMSStore{}
+
+type S3KMSStore struct {
+	svc    s3iface.S3API
+	stsSvc *sts.STS
+	bucket string
+}
+
+// Deprecated; use NewS3KMSStoreWithBucket instead
```

Comment:
You can just delete this whole function I think :)

Reply:
👍